packages/ember/package.json

```diff
@@ -17,7 +17,7 @@
     "test": "tests"
   },
   "scripts": {
-    "build": "ember build --environment=production",
+    "build:uncached": "ember build --environment=production",
```

Comment:
Note: I'll look into actually caching this. For now, it wasn't cached, so no need to cache it anyhow.